### ISSUE TITLE
Design a dark mode

### ISSUE THREAD

**Issue by jveillet** [jveillet/jk-demainilpleut#407]
As a user, I would like to be able to read the site taking into account my preferences for light/dark colors.

* Use the system preferences of the user
* Add an icon + button so that the theme can be switched manually, and on-demand.
* Modify the colors of the site to have better readability on a black background.
* The color of the light theme might have to be also changed depending of the accessibility.

**Resources:**
* https://codesandbox.io/s/dark-mode-toggle-phwrb?fontsize=14&hidenavigation=1&theme=dark
* https://dev.to/ananyaneogi/create-a-dark-light-mode-switch-with-css-variables-34l8
* https://dev.to/corentints/easiest-way-to-do-a-dark-theme-with-css-vars-and-js-26ah
* https://hankchizljaw.com/wrote/create-a-user-controlled-dark-or-light-mode/
